cgroupstats.py

```diff
@@ -77,13 +79,22 @@ class CgroupstatsClient(object):
 
     def __init__(self, verbose=False):
         self.verbose = verbose
-        self.nlsock = netlink.NetlinkSocket()
-
-    def get_load_stats(self, fd):
-        data = CgroupstatsCmdAttrList(cgroupstats_cmd_attr_fd=fd)
-        message = CgroupstatsMessage(
-            'GET', flags=netlink.MessageFlags.REQUEST,
-            attr_list=data,
-        )
-        replies = self.nlsock.query(message)
-        return replies[0].get_attr_list().get('CGROUP_STATS')
+        self.nlsock = netlink.NetlinkSocket(verbose=verbose)
+
+    @contextmanager
+    def open_fd(self, path):
+        fd = os.open(path, os.O_RDONLY)
+        try:
+            yield fd
+        finally:
+            os.close(fd)
+
+    def get_load_stats(self, path):
+        with self.open_fd(path) as fd:
```

Comment:
You don't need to re-implement the context manager here. I believe:
```
with open(path, 'r') as f:
  data = CgroupstatsCmdAttrList(cgroupstats_cmd_attr_fd=f.fileno())
  ...
  ...
```

will do it just fine.